src/main/java/de/hysky/skyblocker/mixins/InGameHudMixin.java

```diff
@@ -156,4 +160,19 @@ public abstract class InGameHudMixin {
     private void skyblocker$afterDrawersInitialized(CallbackInfo ci) {
         this.layeredDrawer.addLayer(HudRenderEvents.LAST.invoker()::onRender);
     }
+
+    @Inject(method = "renderPlayerList", at = @At("HEAD"))
+    private void skyblocker$renderHud(CallbackInfo ci, @Local(argsOnly = true) DrawContext context) {
+        if (!Utils.isOnSkyblock()) return;
+        MinecraftClient client = MinecraftClient.getInstance();
+
+        if (client.currentScreen instanceof WidgetsConfigurationScreen) return;
+        Window window = client.getWindow();
+        float scale = SkyblockerConfigManager.get().uiAndVisuals.tabHud.tabHudScale / 100f;
+        MatrixStack matrices = context.getMatrices();
+        matrices.push();
+        matrices.scale(scale, scale, 1.F);
+        ScreenMaster.render(context, (int) (window.getScaledWidth() / scale), (int) (window.getScaledHeight() / scale));
+        matrices.pop();
+    }
```

Comment:
Why is this a new mixin instead of the old one in PlayerListHudMixin? I don't see a reason.

Reply:
I don't really remember to be honest, but i'm sure i had a good reason